manifests/init.pp

```diff
@@ -316,6 +323,7 @@
   ~> class { '::foreman::install': }
   ~> class { '::foreman::config': }
   ~> class { '::foreman::database': }
+  ~> class { '::foreman::tasks_service': }
```

Comment:
I meant that you should remove this line here and add the contain to ```manifests/service.pp```, but wasn't very clear in that.